dist/tools/flake8/check.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+#
+# Copyright (C) 2017 Alexandre Abadie <alexandre.abadie@inria.fr>
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+if tput colors &> /dev/null && [ $(tput colors) -ge 8 ]; then
+    CERROR="\e[1;31m"
+    CWARNING="\e[1;33m"
+    CRESET="\e[0m"
+else
+    CERROR=
+    CWARNING=
+    CRESET=
+fi
+
+. ${RIOTBASE:+./}dist/tools/ci/changed_files.sh
```

Comment:
Every makefiles is using $(RIOTBASE)/ so I think the '/' can be put after, but maybe I'm wrong.
You could add a DIST_TOOLS variable so the${RIOTBASE} with default value is only done once in the file.

Reply:
After testing a bit, putting the `/` outside was giving `/dist/tools/...` with an empty `RIOTBASE`. The problem was the `+` that should be a `-`:
```
${RIOTBASE:-.}/dist/tools/
```
works when RIOTBASE is empty and also when RIOTBASE is a valid path.

Reply:
And there is the same problem in the ccpcheck.sh script I think